### PR TITLE
Addressed Greg's Additional Technical Comments

### PR DESCRIPTION
**Pull request by yhan8**

### CODE REVIEW

File: 3.resistance-signature/scripts/nbconverted/8.1.last-batch-compile-otherclone-datase.py [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+#remove other clone types from the df
+full_df_val = pd.concat(full_df_val, axis="rows", sort=False).reset_index(drop=True)
+tx_clones = [
+    "TX clone 2",
+    "TX clone 3",
+    "TX clone 4",
+    "TX clone 5",
+    "TX clone 6",
+]
+full_df_val = full_df_val.query("Metadata_clone_number not in @tx_clones")
```
Comment (gwaygenomics) [inline 3.resistance-signature/scripts/nbconverted/8.1.last-batch-compile-otherclone-datase.py:114]: woo!

Comment (gwaygenomics) [reply 3.resistance-signature/scripts/nbconverted/8.1.last-batch-compile-otherclone-datase.py:114]: more pythonic for sure!

File: 3.resistance-signature/scripts/nbconverted/8.compile-otherclone-dataset.py [hunk 2]
```diff
@@ -114,21 +120,33 @@
...
+
+
+
+
+
+# In[ ]:
+
+
+
+
```
Comment (gwaygenomics) [inline 3.resistance-signature/scripts/nbconverted/8.compile-otherclone-dataset.py:152]: This is what happens with extra new lines :)

**Review by gwaygenomics — CHANGES_REQUESTED** [review]
Looks great @yhan8 - please clean up the required elements (delete new lines, that's all) in a couple scripts and lets your this contribution merged in!

File: 2.describe-data/1.merge-datasets-gct.ipynb [hunk 3]
```diff
@@ -969,6 +969,20 @@
...
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
```
[the remark this answers is not in the document]
Comment (yhan8) [reply 2.describe-data/1.merge-datasets-gct.ipynb:985]: Thanks Greg, all set :)

**Pull request merged**